docs/config.md

```diff
@@ -5,15 +5,28 @@ These are the config settings available to jj/Jujutsu.
 
 ## Config files and TOML
 
-The config settings are loaded from the following locations. Less common ways to
-specify `jj` config settings are discussed in a later section.
+`jj` loads several types of config settings:
 
-* [The user config file]
-* `.jj/repo/config.toml` (per-repository)
+- The built-in settings. These cannot be edited. They can be viewed in the
+  `cli/src/config/` directory in `jj`'s source repo.
```

Comment:
I'm tempted to suggest `jj config list --include-defaults` here. The potential problem is that it also includes non-default values. It'll be clearer once we have `--show-origin`. Up to you whether we mention anything about it.

Reply:
I did think along similar lines, and also decided that it'd be clearer if `--show-origin` existed. I also find that `jj config list` is somewhat unpolished, especially for keys that contain spaces, which we do have in the built-in config.

The discussion at https://github.com/martinvonz/jj/pull/2907#discussion_r1475152614 is slightly related.